packages/ns-flashstart/files/flashstart-auth

```diff
@@ -27,6 +29,8 @@ for wan in utils.get_all_wan_devices(u):
     else:
         id = user
     url = f"https://ddns.flashstart.com/nic/update?hostname=&myip=&wildcard=NOCHG&username={id}&password={password}"
-    cmd = ["curl", "-s", "-o", "/dev/null", "--interface", wan, url]
-    subprocess.run(cmd)
+    cmd = ["curl", "-s", "--connect-timeout", "10", "--interface", wan, url]
+    res = subprocess.run(cmd, capture_output=True, text=True)
+    if res.returncode != 0:
+        syslog.syslog(syslog.LOG_ERR, f'Failed to update IP for WAN {wan} -> {id} with error: {res.stdout}')
```

Comment:
```suggestion
        syslog.syslog(syslog.LOG_ERR, f'Flashstart: failed to update IP for WAN {wan} -> {id} with error: {res.stdout}')
```

Reply:
`flashstart-auth` is in the row as the "title" of the log, I can still add this anyway

Reply:
I didn't get it, no need to add anything else